### ISSUE TITLE
In unreleased AG Pro v2.0.0 (124), first directly entering "filters" in Safari Content Blocking, while doing "search" above filters, the color of "cancel" is dark, not gray.

### ISSUE THREAD

**Issue by lancelot-moon** [AdguardTeam/AdguardForiOS#598]
https://youtu.be/u-cYIFpHrXo

Reproduction steps:
Don't need to enable AdGuard VPN.

1. Open AdGuard Pro v2.0.0 (124).
2. First directly enter "Filters" in Safari Content Blocking.
3. Do "search" above those filters. The "cancel" is dark, not gray.

**Comment by zebrum** [comment]
Let's have one issue (#597) for all 'cancel'-button color problem.

**Comment by lancelot-moon** [comment]
@zebrum
@Ivanlin

Zebrum, you should not close this issue.
Different reproduction steps in another place about "cancel".
The issue isn't solved in unreleased AG Pro v2.0.0 (128).

**Comment by lancelot-moon** [comment]
@ameshkov 
The issue was forgotten to be solved in v2.0.0 (128).
May you reopen it again?

**Comment by zebrum** [comment]
@lancelot-moon you are right! For some reason I wasn't able to reproduce the issue last time I tried.

**Comment by IvanIin** [comment]
fixed
